ui/initial_state.py

```diff
@@ -75,6 +83,15 @@ def to_state_file(book_path):
             current_book = main_state['current_book']
             if not current_book == manual_filename:
                 current_book = os.path.join(path, current_book)
+        if 'current_language' in main.state:
+            current_language = main_state['current_language']
+        else:
+            current_language = 'en_GB:en'
+    else:
+        current_language = 'en_GB:en'
+
+    print("current language")
+    print(current_language)
```

Comment:
I have been following a convention of only committing  `log.debug` statements, never `print`. I do use `print` for debugging but don't commit them. I think following this convention will make it harder for random `prints` to accidentally slip into a release. `log.debug` can remain in the code though they do get cleaned up sometimes as well.

Reply:
Yeah, this is only an intermediary commit.  I'm not expecting to have these in the PR when I request the pull